<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SymmetryBrowserDialog</class>
 <widget class="QDialog" name="SymmetryBrowserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Symmetries</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidgetSymmetry">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="CoPlanarity">
      <attribute name="title">
       <string>CoPlanarity</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListWidget" name="listWidgetCoPlanarity"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Reflection">
      <attribute name="title">
       <string>Reflection</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidgetReflection"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Rotation">
      <attribute name="title">
       <string>Rotation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listWidgetRotation"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Translation">
      <attribute name="title">
       <string>Translation</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="listWidgetTranslation"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SymmetryBrowserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SymmetryBrowserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
